src/components/Month.tsx

```diff
@@ -1,13 +1,13 @@
 const Day = (props: { className?: string; contents: string }) => {
-  return <div className={`${props.className || ''} col-1 inline-block h-6 w-6`}>{props.contents}</div>;
+  return <div className={`${props.className || ''} col-1 inline-block h-6 w-6 text-sm`}>{props.contents}</div>;
 };
 
 const DayNumbered = (props: { number: number }) => {
-  return <Day contents={props.number.toString()} />;
+  return <Day className="text-xs cursor-pointer hover:bg-gray-300 rounded-md justify-items-center pt-1 pb-1 pl-1.5 pr-1.5" contents={props.number.toString() } />;
```

Comment:
Alphabetise please

pt-1 and pb-1 can be removed and replaced with py-1 (padding top and bottom 1)

pl-1.5 and pr-1.5 can be removed and replaced with px-1.5 (padding left and right 1.5)

Add the class "select-none" - this prevents the user from dragging over to make the text highlight blue